src/models/options.ts

```diff
@@ -0,0 +1,11 @@
+export interface Options {
+    organisms: optionObject[],
+    annotations: optionObject[],
+    with: optionObject[]
+}
+
+export interface optionObject {
```

Comment:
1. all types and interfaces should be PascalCase
2. keep the file name the same as the Main interface (`Options.ts` in your case)